TidepoolKit/TLogging.swift

```diff
@@ -42,7 +42,7 @@ public protocol TLogging {
 public class TSharedLogging {
 
     /// A shared, global instance of TLogging.
-    public static var instance: TLogging?
+    public static weak var instance: TLogging?
```

Comment:
Who is supposed to set this?  Seems like an odd contract to have a singleton that doesn't hold a ref to this.  Now, anyone setting this like this:
```
class MyLogging: TLogging { ... }
TSharedLogging.instance = MyLogging()
```

will wind up losing their Logger.

Reply:
Fixed. Logger is now a simple property on the `TAPI` object.